app/views/users/index.html.erb

```diff
@@ -1,11 +1,5 @@
 <div class="users-bloc">
-  <% for i in 0..9 do %>
-    <div class="user-bloc">
-      <div class="user-image"></div>
-      <div class="user-data">
-        <h4 class="title">User name</h4>
-        <span class="user-posts">Number of posts: x</span>
-      </div>
-    </div>
+  <% @users.each do |user| %>
+    <%= render partial: 'user', locals: {user: user} %>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169421137-6f38fd3b-6925-4b78-8a5f-2b937edd72dc.png)


- [ ] kindly make sure when I click on the user it should bring me to the `user#show` template/view